src/disco/css/Addon.scss

```diff
@@ -8,23 +8,33 @@ $secondary-text-color: #6a6a6a;
   overflow: hidden;
 
   .theme-image {
+    display: block;
     width: 100%;
   }
 
+  .install-button {
+    margin: 0 20px 0 30px;
+  }
+
   .content {
     align-items: center;
     display: flex;
     flex-direction: row;
     line-height: 1.5;
-    padding: 20px;
 
     .logo {
-      height: 64px;
-      width: 64px;
+      background: #fcfcfc;
```

Comment:
Ah good catch - the terrible contrast makes this background almost invisible to me. Using `align-self: stretch` on `.logo` and resetting the centering for just the `img` was a succinct solution.